rts/at/sra/README.md

```diff
@@ -0,0 +1,55 @@
+OVERVIEW
```

Comment:
I haven't fully reviewed the PR yet but I wanted to add a few comments about the README before doing so:

Did you say that you had already done most of work for https://issues.redhat.com/browse/JBTM-3463 (splitting up the services), if so you will need to rewrite the README because the instructions will change, quite substantially. Perhaps you should write it once and fix both issues in the same PR.

Secondly, the format we use for the other REST-AT quickstarts is:

```
OVERVIEW
USAGE
EXPECTED OUTPUT
WHAT JUST HAPPENED?
```

It might seem a bit clunky but that is the format used in the other six REST-AT quickstarts so best to remain consistent so as to present a better user experience.

Also I would use the term REST-AT annotations or something similar rather than SRA which was introduced while making a proposal to the Eclipse MicroProfile group to provide client side annotations for controlling the lifecycle of a REST-AT transaction (it was a complement to Long Running Actions). Since the SRA component of the proposal was rejected and the code has been moved into the narayana repo, we need to use the term REST-AT annotations since SRA will confuse readers. As an aside, the directory name `sra` should be replaced with something more meaningful in the context of REST-AT, I would suggest `annotation`.

Reply:
@mmusgrov I've updated the README, please review it.
Thanks.